apps/web/src/contexts/nftPreviewFallback/NftPreviewFallbackContext.tsx

```diff
@@ -0,0 +1,80 @@
+import { createContext, ReactNode, useCallback, useContext, useMemo, useState } from 'react';
+
+export type Dimensions = {
+  width?: number;
+  height?: number;
+};
+
+export type CacheParams = {
+  [tokenId: string]: {
+    type: 'preview' | 'raw';
```

Comment:
`'preview' | 'raw'` is repeated a few times here. let's assign it to its own type, like `LoadedNftAssetType`, and use that variable everywhere (including L51)